### DISCUSSION TITLE
Safari Cross Site Tracking preventing cookie and log-in

### DISCUSSION THREAD

**Discussion by steven-js** [keystonejs/keystone#6127] [Event Q&A Q3 2021]
Hi 👋 the Keystone guys know about this on Slack but I'm adding it here too just incase anyone has been thinking about a work around 🤔 … 

Unless we uncheck Safari’s Cross Site Tracking, we can't log-in through the Safari browser and no response cookie.

Has anyone thought of a way to tackle this?

<img width="1332" alt="screenshot_2021-07-14_at_07 19 22" src="https://user-images.githubusercontent.com/35059438/125922496-e8c985de-348a-4614-9dfc-4462381b0f25.png">
<img width="919" alt="Screenshot 2021-07-16 at 09 58 47" src="https://user-images.githubusercontent.com/35059438/125922503-77aab354-9132-44ef-b969-a64b318ef7cc.png">

**Comment by bladey** [comment]
This has now been solved through the Keystone Community Slack - https://keystonejs.slack.com/archives/C01STDMEW3S/p1631214630398900?thread_ts=1626082293.166300&cid=C01STDMEW3S
